app.js

```diff
@@ -1,50 +1,40 @@
-$(document).on("click", function(event) {
+//Initialize function when user clicks anywhere on the document
+$(document).on("click", function() {
 
     // Here we grab the text from the input box
     // var pokemon = $("#pokeMon").val();
```

Comment:
Why is this commented out?

Reply:
Initial testing process that was not yet removed, will be removed in next push!